### ISSUE TITLE
Assets pulling from http instead of https

### ISSUE THREAD

**Issue by zedee** [RoumenDamianoff/laravel-assets#27]
I'm developing in testing environment, and having SSL enabled (my protocol is https instead of http).

I'm noticing the assets are loaded from ```http:``` instead of ```https:```

We add the assets on a middleware that runs before the request. In there, we've set the ```setDomain``` to https://ourtesturl.local , and dumping the ```$domain``` variable seems to be set correctly to ```https```, however the assets are requested from ```http``` instead. 

I also checked out it wasn't a ```.htaccess``` misconfiguration, but even forcing to rewrite, we cannot load the assets from our local https.

Any ideas?

**Comment by ety001** [comment]
I have the same problem.

**Comment by max-kk** [comment]
You need use following code in order to use HTTPS instead HTTP
```php
   \Asset::$secure = true;
```

**Comment by zedee** [comment]
Yes, this does work indeed. So is that the proper way to go then?

**Comment by max-kk** [comment]
Yes, I added this in one of my middleware class to make it always ON.

**Comment by itelmenko** [comment]
Can somebody past example to here?

**Comment by max-kk** [comment]
Example:

```
<?php

namespace App\Http\Middleware;

use Closure;

class SessionGroupMiddleware
{
    /**
     * Handle an incoming request.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \Closure  $next
     * @return mixed
     */
    public function handle($request, Closure $next)
    {
        if (env('APP_ENV') !== 'local') {
           \Asset::$secure = true;
        }

        // TODO - REMOVE
        \Debugbar::disable();

        return $next($request);
    }
}
```

**Comment by itelmenko** [comment]
I have got an error with test
`Class 'Asset' not found`

**Comment by itelmenko** [comment]
> Any ideas about what we're missing or doing incorrectly?

May be your server does not send $_SERVER[https] variable in php?

Becase in ../vendor/symfony/http-foundation/Request.php we have method isSecure().

In this method we have this code:

`
...

$https = $this->server->get('HTTPS');

return !empty($https) && 'off' !== strtolower($https);
`

It is for Laravel 5.4
